### PR TITLE
Various cleanups

### PR DESCRIPTION
**Pull request by xsawyerx**
Check each commit separately.

### CODE REVIEW

File: lib/Pakket/Repository/Backend/DBI.pm [hunk 1]
```diff
@@ -57,53 +95,64 @@ sub retrieve_location {
...
+            $DBI::errstr,
+        );
+
         exit 1;
     }
 
     my $all_content = $stmt->fetchall_arrayref();
-    if (!$all_content || @$all_content != 1) {
-        $log->criticalf('Failed to retrieve exactly one row for id %d: %s', $id);
+    if ( !$all_content || @{$all_content} != 1 ) {
```
Comment (mickeyn) [inline lib/Pakket/Repository/Backend/DBI.pm]: clearer logic:
`if ( is_arrayref($all_content) and @{$all_content} != 1 ) {`

Comment (mickeyn) [inline lib/Pakket/Repository/Backend/DBI.pm]: corrected original post, but still this is clearer

File: lib/Pakket/Repository/Parcel.pm [hunk 2]
```diff
@@ -50,8 +33,7 @@ sub store_package_parcel {
     if ( $parcel_path->is_file ) {
         # We were given a file, which we assume is a valid tar file.
         $file = $parcel_path;
-    }
-    elsif ( $parcel_path->is_dir ) {
+    } elsif ( $parcel_path->is_dir ) {
```
Comment (mickeyn) [inline lib/Pakket/Repository/Parcel.pm]: I don't understand this change, I don't see this as clearer style (that's subjective) and prefer not to see such changes.

File: lib/Pakket/Repository/Parcel.pm [hunk 3]
```diff
@@ -76,20 +58,19 @@ sub store_package_parcel {
         # Write and compress
         $log->debug("Writing archive as $file");
         $arch->write( $file->stringify, 1 );
-    }
-    else {
+    } else {
```
Comment (mickeyn) [inline lib/Pakket/Repository/Parcel.pm]: same

File: lib/Pakket/Repository/Source.pm [hunk 4]
```diff
@@ -50,8 +33,7 @@ sub store_package_source {
     if ( $source_path->is_file ) {
         # We were given a file, which we assume is a valid tar file.
         $file = $source_path;
-    }
-    elsif ( $source_path->is_dir ) {
+    } elsif ( $source_path->is_dir ) {
```
Comment (mickeyn) [inline lib/Pakket/Repository/Source.pm]: same

File: lib/Pakket/Repository/Source.pm [hunk 5]
```diff
@@ -76,32 +58,19 @@ sub store_package_source {
         # Write and compress
         $log->debug("Writing archive as $file");
         $arch->write( $file->stringify, 1 );
-    }
-    else {
+    } else {
```
Comment (mickeyn) [inline lib/Pakket/Repository/Source.pm]: same

File: lib/Pakket/Repository/Backend/DBI.pm (hunk 1 above)
Comment (xsawyerx) [inline lib/Pakket/Repository/Backend/DBI.pm]: I just cleaned it up a bit. I can add a commit to also change the logic test, but I think is_arrayref() is unnecessary here. It also makes Pakket unfatpackable, I see.

File: lib/Pakket/Repository/Parcel.pm (hunk 2 above)
Comment (xsawyerx) [inline lib/Pakket/Repository/Parcel.pm]: We have a perltidy that we are using (or to rephrase, that we agreed to use, but only I'm using) and I want to maintain at least similar code. This is the *only* place in which we have uncuddled conditions. Does it really bother you this much to change it to look like all the rest of our code?

File: lib/Pakket/Repository/Parcel.pm (hunk 3 above)
Comment (xsawyerx) [inline lib/Pakket/Repository/Parcel.pm]: Same.

File: lib/Pakket/Repository/Source.pm (hunk 4 above)
Comment (xsawyerx) [inline lib/Pakket/Repository/Source.pm]: Same.

File: lib/Pakket/Repository/Source.pm (hunk 5 above)
Comment (xsawyerx) [inline lib/Pakket/Repository/Source.pm]: Same.

**Comment by xsawyerx** [comment]
Thank you for entertaining my annoying OCD. :)

**Pull request merged**